README.md

```diff
@@ -1 +1,25 @@
 # Hello-rails-react
+
+Implement a connection between a Ruby on Rails back-end and React front-end. Understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
+
+  * Built With
+  * Ruby 3.0.3
+  * Rails 6.1.6
+
+Author
+👤 Firdavs
+
+GitHub: @fed1k
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
```

Comment:
- [ ] Please update your readme file using the [Micrverse Readme Template](https://github.com/microverseinc/readme-template)